src/configuration.js

```diff
@@ -177,8 +177,7 @@ function loadOptionsFromConfig(configDirectory) {
 
   const cosmic = cosmiconfig('graphql-schema-linter', {
     cache: false,
-    sync: true,
-  }).load(searchPath);
+  }).searchSync(searchPath);
```

Comment:
Note to self about this change:

> Additionally, the overloaded load() function has been split up into several clear and focused functions:
> 
> search() now searches up the directory tree, and load() loads a configuration file that you don't need to search for.
> The sync option has been replaced with separate synchronous functions: searchSync() and loadSync().

From [CHANGELOG.md](https://github.com/davidtheclark/cosmiconfig/blob/f1750cde0b9c53bc577efa60d012a85a4dc4b823/CHANGELOG.md#501)